Example/ParalayoutSnapshotTests/ViewDistributionSnapshotTests.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  Copyright © 2021 Square, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Paralayout
+import SnapshotTesting
+
+final class ViewDistributionSnapshotTests: SnapshotTestCase {
+
+    func testSpreadSubviews() {
+        let container = UIView(frame: CGRect(x: 0, y: 0, width: 400, height: 100))
+        container.backgroundColor = .white
+
+        let redView = UIView()
+        redView.backgroundColor = .red
+        container.addSubview(redView)
+
+        let blueView = UIView()
+        blueView.backgroundColor = .blue
+        container.addSubview(blueView)
+
+        let greenView = UIView()
+        greenView.backgroundColor = .green
+        container.addSubview(greenView)
+
+        func verifySnapshot(
```

Comment:
I set this up planning to have more calls to it with different combinations, but found a few bugs as I was testing it (#58 and #59). We should add those combinations as we resolve the issues.